projects/core/src/occ/utils/occ-http-token.ts

```diff
@@ -0,0 +1,20 @@
+/*
+ * SPDX-FileCopyrightText: 2022 SAP Spartacus team <spartacus-team@sap.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { HttpContextToken } from '@angular/common/http';
+
+export interface OccHttpContext {
+  /**
+   * Indicates to the interceptor that the emulated customer ID should be sent to the OCC
+   * to avoid ambiguity.
+   * 'string' if the request has the user ID at hand.
+   * 'boolean' (or, 'true') if the request does not have the user ID but still needs to send the emulated ID.
+   * @see feature-libs/asm/root/interceptors/user-id-http-header.interceptor.ts
```

Comment:
I think this comment is also generated and published online, in which case the file path won't mean much.  Perhaps we use the class name and import path to refer to other classes.

Reply:
I can remove this line, but I'd like to see where this type of documentation gets published. This is an unusual obstacle for writing notes.
If the solution is to remove the JSDoc tag, that works for me.

Reply:
It's [this doc](https://sap.github.io/spartacus/injectables/AsmService.html), where each element of the Public API is documented.

However, I just saw that _all_ the api elements in this doc are referenced from their source code file path, so forget about my objections.